aws/provider_test.go

```diff
@@ -750,3 +749,21 @@ data "aws_arn" "test" {
 }
 `, region)
 }
+
+func testAccAssumeRoleARNPreCheck(t *testing.T) {
+	v := os.Getenv("TF_ACC_ASSUME_ROLE_ARN")
+	if v == "" {
+		t.Skip("skipping tests; TF_ACC_ASSUME_ROLE_ARN must be set")
```

Comment:
Nit: In the future, we may want to augment this skip message to include more details about what this is and how to create it, e.g.

```suggestion
		t.Skip("skipping tests; TF_ACC_ASSUME_ROLE_ARN environment variable must be set.\n" +
			"This acceptance test expects the ARN of an IAM Role with full permissions " +
			"in the testing AWS account that can be assumed with a restrictive session " +
			"policy to verify resource functionality with those restrictive permissions.")
```